test/cpp/api/jit.cpp

```diff
@@ -99,6 +100,19 @@ TEST(TorchScriptTest, TestDictArgMatching) {
   ASSERT_EQ(1, output.toTensor()[0].item<int64_t>());
 }
 
+
+TEST(TorchScriptTest, Conversion_MultiCUDA) {
```

Comment:
The test should be in `cpp/jit/`, not `api/jit.cpp`, see `testModuleDefine` to see an example of a manually added parameter (attributes have pretty much the same API on modules)